css/sass/dialog.scss

```diff
@@ -197,7 +213,11 @@ $object-field-background: #ebf1f5;
 }
 
 .gohan-form-object-children {
-  padding-left: 10px;
+  // padding-left: 10px;
```

Comment:
IMHO this padding helps user to see hierarchy in object. Without it, user may not know at once which fields are together in one object/group :slightly_smiling_face:

Reply:
I got your point, I fixed it. Please confirm.